nixos/modules/security/auditd.nix

```
@@ -4,16 +4,260 @@
   pkgs,
   ...
 }:
+let
+  cfg = config.security.auditd;
 
+  audit_config_type =
+    with lib.types;
+    nullOr (oneOf [
+      bool
+      nonEmptyStr
+      path
+      int
+    ]);
+
+  plugin_conf = lib.types.submodule {
```

Naming is inconsistent and [should follow snakeCase](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax) 

Also, this should probably be named something like the following to be a little more consistent with the rest of Nixpkgs:

```suggestion
  pluginOptions = lib.types.submodule {
```